src/apis/index.ts

```diff
@@ -16,7 +17,9 @@ export const auth = new Elysia({
   detail: {
```

Comment:
let's add this prefix to `ENDPOINT` constant. That is similar `theme` controller